src/gan/querygan_pyt.py

```diff
@@ -142,8 +161,10 @@ def generate_samples(net, seq_len, generated_num, parser, oracle=False,
                 parfile.write(' '.join([str(x) for x in parents]) + '\n')
                 tokfile.write(' '.join(tokens) + '\n')
                 catfile.write('0' + '\n')
-    
-    return zip(samples, examples)
+
+        return None

```

Comment:
Again, this concept of returning a different number of arguments is not best practice.